### PR TITLE
Ge upd donation builder copy 1028

### LINKED ISSUE #1028: Update donation builder copy
**User Story:**
As a user
I want to make sure my donation counts towards the matching calculation
So that the projects I support can benefit

**Acceptance Criteria**
- Adding one line of copy under the "Submit your donation" button

**Product & Design Links:**
[Figma](https://www.figma.com/file/lZwHrKYiWJna3AePzyTAiG/Grant-Explorer?node-id=3180%3A85249&t=GbhISJ168BQoz6ps-4)

**Notes/Assumptions:**
- This includes writing test coverage for the functionality added in the ticket, if necessary

### PR DESCRIPTION
**Pull request by DanieleSalatti**
Fixes #1028

### CODE REVIEW

**Review by melissa-neira — APPROVED** [review]
Approved on the design side but some tests seem to be failing

**Comment by bhargavaparoksham** [comment]
@DanieleSalatti some tests seem to be failing, can you fix them as well?

**Comment by nfrgosselin** [comment]
Hmm... I'm not seeing a fleek preview link for GE?

**Comment by melissa-neira** [comment]
@nfrgosselin [here you go](https://bafybeiaczgjn4jitp55aj7627wzwprjuwrwylr56jaeosyd4nnhpvegrwu.on.fleek.co/) 
if you go to the details page for GE and scroll down you'll see a hyperlink that says "View more details on Fleek.co" and it was the preview link 😎

**Review by nfrgosselin — APPROVED** [review]

**Comment by nfrgosselin** [comment]
Ahaaa thanks @melissa-neira! Just approved from Product. Defer to eng on fixing the failing tests

File: packages/grant-explorer/.env.sample [hunk 1]
```diff
@@ -19,7 +19,7 @@ REACT_APP_DATADOG_SERVICE=######################
 REACT_APP_DATADOG_SITE=######################
 
 # Sentry Monitoring
-REACT_APP_SENTRY_DSN==######################
+REACT_APP_SENTRY_DSN=######################
```
Comment (thelostone-mc) [inline packages/grant-explorer/.env.sample:22]: oh lol nice catch

**Review by thelostone-mc — APPROVED** [review]

**Pull request merged**